decentraland/unity-renderer
# refactor: friend requests

## What does this PR change?

Fixes the technical debt suggested in #3960.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=refactor/friend-requests
2. Check that the friend requests flows keep working as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
